src/components/lyne-cta-button/lyne-cta-button.tsx

```diff
@@ -24,6 +25,9 @@ export class LyneCtaButton {
   /** Label text to show on the button */
   @Prop() public label = 'Default button text';
 
+  /** Type of the button, like primary, secondary etc. */
+  @Prop() public type?: InterfaceButtonAttributes['type'];
```

Comment:
I'm wondering, whether we should name this 'variant' or something similar, since type is also used an official attribute already (button, submit, reset).